app/controllers/nonsipp/bondsdisposal/TotalConsiderationSaleBondsController.scala

```diff
@@ -44,7 +45,7 @@ class TotalConsiderationSaleBondsController @Inject()(
   saveService: SaveService,
   @Named("non-sipp") navigator: Navigator,
   identifyAndRequireData: IdentifyAndRequireData,
-  formProvider: MoneyFormErrorProvider,
+  formProvider: MoneyFormProvider,
```

Comment:
Should we do this change in .g8 files too and remove MoneyFormErrorProvider?

Reply:
I think that's a good idea, unless there are concerns it might break things, or incur significant re-testing efforts to include that in this ticket. Or would this change be done as part of a separate branch/ticket?

Reply:
Yes, in the follow up branch with unit tests please.